.rubocop.yml

```diff
@@ -1,31 +1,14 @@
-require:
-  - rubocop-performance
-  - rubocop-rails
-  - rubocop-minitest
+inherit_gem:
+  theforeman-rubocop: all.yml
 
 inherit_from: .rubocop_todo.yml
 
-AllCops:
-  TargetRubyVersion: 2.5
```

Comment:
Does it not inherit this from all.yml? Looks like it's falling back to 2.1 now.

Reply:
It should, I think it was just picking up on a wrong gem version.